### ISSUE TITLE
`disallowed-unique-constraint` shouldn't trigger on new columns

### ISSUE THREAD

**Issue by reteps** [sbdchd/squawk#649]
```sql
ALTER TABLE course_instances
ADD COLUMN enrollment_code VARCHAR(255) UNIQUE;
```

I don't think this rule should trigger.

**Comment by reteps** [comment]
> Currently we have it in the docs that it's blocking but I'll have to double check.

If there is no values in the column, it should succeed instantly AFAIK.

**Comment by reteps** [comment]
Ah -- soryy about that!
